src/06-positron-rag/lib/extract/transformArticle.ts

```diff
@@ -37,6 +40,15 @@ function getArticleBody(article: any) {
   return text
 }
 
+function getArticleSections(article: any) {
+  const sections = article.sections
```

Comment:
I realized that Positron's modeling of articles as a collection of `section`s might actually be a boon instead of a liability — it gives us _natural_ chunks of text, so that we don't have to devise a chunking strategy of our own.

So in addition to the the fulltext that I already had by concatenating these sections (L38-39 above), I also now retain the individual sections.

(I don't do anything with the chunks in this PR, but that would be the next obvious thing to try for RAG)

Reply:
> but that would be the next obvious thing to try for RAG

Is the idea that we may only care about the text in the section, and not the rest of the article, so this could allow us to not over-fetch?